### ISSUE TITLE
Forced width/height fixed for mobile platforms

### ISSUE THREAD

**Issue by jgranick** [openfl/openfl-native#108]
#108 Issue by __restorer__,

**Comment by jgranick** [comment]
Comment by __jgranick__:

Hi!

I'm not quite sure I understand what this change does. Is this to prevent the title from being scaled while running on a mobile platform, or does this fix a scenario where the title is scaled, but later, changes in the game/application cause it to scale improperly?

Thanks so much!

**Comment by jgranick** [comment]
Comment by __restorer__:

Hi!

Sorry for so long description, but I try my best describe the problem.

As I understand, in the `project.xml` I can put `<window width="xxx" height="yyy" />` to set exact window size.
This work well for desktop (neko, osx, etc) and for flash.
But this feature didn't work as expected for mobile platforms. For example I set `<window width="320" height="200" />`. I expect that now my game will work in virtual resolution 320x200 and virtual stage will be scaled automatically to   real screen size. But this doesn't happen. I take some screenshots to illustrate the problem:

Standard sample (DisplayingABitmap) - http://images.detstwo.com/view/4LnuOU.png (looking good, real device resolution is 1280x720, image size is 400x400).

I put `<window width="320" height="240" background="#800000" resizable="false" orientation="landscape" />` into project.xml

What I expect - http://images.detstwo.com/view/RNc76x.png (black bars at the left and right - because ratio 1280:720 != ratio 320:240, image cropped because it is bigger than 320x240)

What I got instead - http://images.detstwo.com/view/an4WaX.png (image partially cropped, at the top-left corned instead of center, and image is not enlarged to fit the screen)

`ApplicationMain.hx` already has code that should fix this (`ScaledStage` class and `applyScale` function), but unfortunately it didn't work as expected.

For example line 122:

```
var xScale:Float = untyped (flash.Lib.current.stage).stageWidth / forceWidth;
```

It should calculate xScale ratio by dividing actual stage width `untyped (flash.Lib.current.stage).stageWidth` by `forceWidth`.
But it didn't work, because in mobile target `ScaledStage` is used instead of original flash stage, and `untyped (flash.Lib.current.stage).stageWidth` is forced width instead of original width. See lines 221-225:

```
private override function get_stageWidth ():Int {

    return ::WIN_WIDTH::;

}
```

Due to this, in function `applyScale` xScale and yScale is always equals to 1.
Maybe I don't understand something, but I think that this is a bug.

About fix:

ScaledStage already has methods to get original stage width and height (__stageWidth and __stageHeight).
So fix is really simple - all we need is to use `scaledStage.__stageWidth` instead of `untyped (flash.Lib.current.stage).stageWidth` (same for height).
